tests/test_iib_build_details_model.py

```diff
@@ -275,8 +313,36 @@ def test_to_dict(fixture_rm_build_details_json):
         distribution_scope="null",
     )
 
-    model = RmModel.from_dict(fixture_rm_build_details_json).to_dict()
-    assert model == rm_model.to_dict()
+    mii_model = MergeIndexImageModel(
```

Comment:
Maybe you could separate this to multiple tests? If more models are added this test would be too long.

Reply:
The test is the same just trying a new model. I think it is not necessary

Reply:
Well it's kind of weird that it only tests 2/4 models, but if it tested all I think it would have been too long

Reply:
Could we use `@pytest.mark.parametrize()` for this? Since as you mentioned the test is the same, just different data.

Reply:
Separated into 2 tests. It is the easiest way because I can't generalize the test since I need to use a particular model e.g. `AddModel(**args)`